feature-libs/quote/components/details/cart/quote-details-cart.component.html

```diff
@@ -1,19 +1,25 @@
-<div class="cart-toggle" (click)="showCart = !showCart">
-  <cx-icon
-    aria-hidden="false"
-    [type]="showCart ? iconTypes.CARET_UP : iconTypes.CARET_DOWN"
-  >
-  </cx-icon>
-  <span class="cart-toggle-text">{{ 'quote.commons.cart' | cxTranslate }}</span>
-</div>
+<ng-container *ngIf="{ showCart: showCart$ | async } as state">
```

Comment:
Why not simply
```suggestion
<ng-container *ngIf="(showCart$ | async ) as showCart">
```

Reply:
in case `showCart=false` the container would not be rendered, hence even the toggle, which is inside would not be rendered, which is not desired.

Reply:
My point: The ngIf in line #1 is misleading, right? Because this will instantly be defined.

Reply:
Just did a bit of searching: Yes, this seems to be the standard way, no other directive available. Even if it's a bit weird. So: ok from my side